packages/table/src/table.tsx

```diff
@@ -41,6 +42,14 @@ import { TableBody } from "./tableBody";
 
 export interface ITableProps extends IProps, IRowHeights, IColumnWidths {
     /**
+     * If `true`, there will be a single "focused" cell at all times,
+     * which can be used to interact with the table as though it is a
+     * spreadsheet.
```

Comment:
needs better docs for `false` case. 

how does the focus cell behave if this is `false`? is it even supported?